packages/cry/cry.0.4.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-cry"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "cry"]
+depends: ["ocamlfind"]
```

Comment:
Is `ocamlfind` only needed for build (in which case you should add `{build}`), or also for its libraries (`bytes` compatibility in particular) ?